bindings/cpp/AndroidSecureStorage/app/src/main/java/com/cronos/play/SecureStorage.kt

```diff
@@ -0,0 +1,91 @@
+package com.cronos.play
```

Comment:
formatting of this file may be a bit strange, you can use ktfmt: https://facebook.github.io/ktfmt/ to format it

Reply:
ok